apps/barista/src/shared/location.service.ts

```diff
@@ -39,6 +39,14 @@ export class BaLocationService {
     map(url => (url.match(/[^?#]*/) || [])[0]), // strip query and hash
   );
 
+  currentQuery$: Observable<URLSearchParams> = this._urlSubject.pipe(
+    map(currentUrl =>
+      currentUrl.includes('?')
```

Comment:
What if the url includes other query params? (like `?param1=some%20string&q=search-term`) you never know ;)